commit 1ee02b3dea0e3d3ac6a00253e8656117a07cca0a
Author: Adrian Knoth <adi@drcomp.erfurt.thur.de>
Date:   Mon Jun 8 18:05:29 2009 +0200

    JACK input support for jack2

    When you try to capture from JACK, this is only possible with the
    "oldstyle" jackd-0.x. If you want to use the newer, SMP-capable jack2,
    the VLC inputs cannot be connected:

    Cannot connect ports owned by inactive clients: "vlc-input-26611" is not
    active
    Cannot connect ports owned by inactive clients: "vlc-input-26611" is not
    active

    Unlike jack1, jack2 expects the process callback function to return 0.

    The attached patch fixes this problem. It's been already fixed the same
    way in the jack output plugin (see modules/audio_output/jack.c:290).

    Taken from Debian bug #532339
    (http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=532339)
The attached patch fixes this problem. It's been already fixed the same
    way in the jack output plugin (see modules/audio_output/jack.c:290).

    Taken from Debian bug #532339
    (http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=532339)

    Signed-off-by: Christophe Mutricy <xtophe@videolan.org>

diff --git a/modules/access/jack.c b/modules/access/jack.c
index 217b6e4..1e08c4c 100644
--- a/modules/access/jack.c
+++ b/modules/access/jack.c
@@ -444,7 +444,7 @@ int Process( jack_nframes_t i_frames, void *p_arg )
         }
     }

-    return 1;
+    return 0;
 }

